app/views/placements/support/organisations/new.html.erb

```diff
@@ -2,19 +2,21 @@
   <%= govuk_back_link(href: placements_support_organisations_path) %>
 <% end %>
 
-<%= form_with(model: @organisation, url: select_type_placements_support_organisations_path, method: "get", data: { turbo: false }) do |f| %>
-  <%= f.govuk_error_summary %>
+<div class="govuk-width-container">
```

Comment:
This breaks the position of the back_link. Possibly, the backlink needs to be wrapped in a div with the 'govuk-width-container' class.
<img width="875" alt="image" src="https://github.com/DFE-Digital/itt-mentor-services/assets/44073106/d4b237f9-845a-4ff7-a7c4-8800d7bcdcb8">

Reply:
😅 I've admittedly been lazy to address it and was hoping to push it into a future PR. But since there are a couple of other things to address, I might as well pick it up.